mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.tmpl

```diff
@@ -601,6 +633,23 @@ resource "google_compute_subnetwork" "network-with-private-google-access" {
 `, cnName, subnetworkName, cidrRange)
 }
 
+func testAccComputeSubnetwork_update4(cnName, subnetworkName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_network" "custom-test" {
+  name                    = "%s"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "network-with-migration-purpose" {
```

Comment:
This test should update the previously existing network instead of deleting it and creating a new one.

```suggestion
resource "google_compute_subnetwork" "network-with-private-google-access" {
```